src/components/DocumentList/VerifiedLabel.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { render } from "@testing-library/react-native";
+import { VerifiedLabel } from "./VerifiedLabel";
+
+jest.mock("@expo/vector-icons", () => ({ Feather: "Feather Icons" }));
+
+describe("VerifiedLabel", () => {
+  it("should show VERIFIED when isVerified is true", async () => {
+    expect.assertions(1);
```

Comment:
Do we need these assertions given that the render isn't async?

Reply:
Nvm, looks like we're using the entire jest eslint plugin.

Reply:
yea...